components/03-modules/filter/_filter.scss

```diff
@@ -0,0 +1,144 @@
+$filter__color-sample-border-radius        : 100% !default;
+$filter__color-label-font-size             : 10px !default;
+$filter__color-label-color                 : $gray-darkest !default;
+// Show-all
+$filter__show-all-height                   : 48px !default;
+$filter__show-all-font-size                : 12px !default;
+// Divider
+$filter__divider-border-top                : solid 1px lighten($gray-lighter, 15%) !default;
+// Range
+$filter__range-margin                      : 0 0 $spacer--medium 0 !default;
+
+.filter {
+    &__label {
+        padding: $filter__label-padding;
+    }
+
+    &__swatch {
+        display: flex;
+        flex-wrap: wrap;
+        margin: $filter__swatch-margin;
+        &-option {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            min-width: $filter__swatch-option-size;
+            min-height: $filter__swatch-option-size;
+            margin-right: $filter__swatch-option-margin;
+            border: $filter__swatch-option-border;
+            background-color: $filter__swatch-option-background;
+            color: $filter__swatch-option-color;
+            transition: $filter__transition;
+            cursor: pointer;
+            text-decoration: none;
+
+            &:hover,
+            &:focus {
+                text-decoration: none;
+                border-color: $filter__swatch-option-border-color-hover;
+            }
+
+            &:last-child {
+                margin-right: 0;
+            }
+
+            &--active {
+                border-color: $filter__swatch-option-border-color--active;
+            }
+        }
+    }
+
+    &__color {
+        display: flex;
+        flex-flow: row wrap;
+        margin: $filter__color-margin;
+
+        &-option {
+            position: relative;
+            display: flex;
+            flex-direction: column;
+            justify-content: flex-start;
+            align-items: center;
+            max-width: $filter__color-option-width;
+            width: $filter__color-option-width;
+            height: $filter__color-option-height;
+            max-height: $filter__color-option-height;
+            text-decoration: none;
+
+            &:after {
+                content: '';
+                position: absolute;
+                top: $filter__color-option-after-top;
+                left: $filter__color-option-after-left;
+                width: $filter__color-option-after-size;
+                height: $filter__color-option-after-size;
+                background-color: transparent;
+                border-radius: $filter__color-option-after-border-radius;
+                transition: $filter__transition;
+            }
+
+            &--active,
+            &:hover,
+            &:focus {
+                text-decoration: none;
+                &:after {
+                    border: $filter__color-option-after-border;
+                }
+            }
+        }
+
+        &-sample {
```

Comment:
the same (element inside another element)

Reply:
Done